docs/docs/core/forms.mdx

```diff
@@ -5,276 +5,402 @@ title: Forms
 description: Follow a rule-based process of information gathering using FormActions in open source bot framework Rasa.
 ---
 
-:::note
-There is an in-depth tutorial [here](https://blog.rasa.com/building-contextual-assistants-with-rasa-formaction/) about how to use Rasa Forms for slot filling.
-
-:::
-
 One of the most common conversation patterns is to collect a few pieces of
 information from a user in order to do something (book a restaurant, call an
 API, search a database, etc.). This is also called **slot filling**.
 
-If you need to collect multiple pieces of information in a row, we recommended
-that you create a `FormAction`. This is a single action which contains the
-logic to loop over the required slots and ask the user for this information.
-There is a full example using forms in the `examples/formbot` directory of
-Rasa Core.
+## Usage
 
-When you define a form, you need to add it to your domain file.
-If your form’s name is `restaurant_form`, your domain would look like this:
+To use forms with Rasa Open Source you need to make sure that the
+[RulePolicy](./policies.mdx#rule-policy) is added to your policy configuration.
+For example:
 
 ```yaml
-forms:
-  - restaurant_form
-actions:
-  ...
+policies:
+- ... # other policies
+- name: RulePolicy
 ```
 
-See `examples/formbot/domain.yml` for an example.
+<a aria-hidden="true" tabIndex="-1" className="anchor enhancedAnchor" id="form-domain"></a>
```

Comment:
not required, docusaurus generates them by default for every heading, I believe

Reply:
why is it then in all other `mdx` files? I quickly skimmed over them and adapt it from there.

Reply:
it came from the automatic generation.

Reply:
ah, ok. So I can remove them all?

Reply:
yeah the one in your doc. But leave the other ones as it could break permalinks